### ISSUE TITLE
quoted-expressions should ignore mixed attribute bindings

### ISSUE THREAD

**Issue by 43081j** [43081j/eslint-plugin-lit#154]
In cases like the following, we should _not_ enforce lack of quotes:

```ts
html`
  <div data-foo="im a ${mixed} value">
  </div>
`;
```

**Comment by 43081j** [comment]
actually looks like it does this already and i got mixed up!

closing
